content/en/docs/v3.4/dev-guide/features.md

```diff
@@ -0,0 +1,43 @@
+---
+title: etcd features
+weight: 3750
+description: using etcd features
+---
+
+This document provides an overview of etcd features to help users better understand the features and related deprecation process. If you are interested in knowing about how features are developed in the etcd, please see these [development guidelines](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/features.md).
+
+The etcd features fall into three stages, experimental, stable, and unsafe. You can get the list of features by running `etcd --help`.
+
+## Experimental
+
+In order to get early feedback, any new feature is usually added as an experimental feature. The experimental feature can be identified by looking at the flag name, which should have `--experimental` as a prefix. Please consider the following points while using an experimental feature: 
+- It might be buggy due to a lack of user testing. Enabling the feature may not work as expected.
+- It is disabled by default. 
+- Support for such a feature may be dropped at any time without notice
+    - It can be removed in the next minor or major release without following the [feature deprecation](#feature-deprecation) policy unless it graduates to a stable future.
+    - The project team would appreciate users reporting any issues related to experimental features. However, such issues may be given lower priorities compared to the issues related to stable featuers.
+- An experimental [feature deprecates](#feature-deprecation) when it graduates to the stable stage. Users should start using a stable feature as soon as possible.
```

Comment:
```suggestion
- An experimental [feature flag deprecates](#feature-deprecation) when it graduates to the stable stage. Users should start using a stable feature flag as soon as possible.
```

Reply:
@ahrtr thanks for the quick review. This depends on how we read it but it sounds good to me this way too. I am updating accordingly.

Reply:
ack.

As an non-native English speaker, previous sentence is a little confusing to me. It looked like the deprecated experimental feature and the graduated stable feature are two different features. But actually they are the same feature, they just have two different flags. 

It's much better now to me. 

Thank.

Reply:
@ahrtr +1 thanks and sounds great!